tests/ProxyManagerTest/ProxyGenerator/PropertyGenerator/AbstractUniquePropertyNameTest.php

```diff
@@ -34,15 +34,15 @@
 abstract class AbstractUniquePropertyNameTest extends PHPUnit_Framework_TestCase
 {
     /**
-     * Verifies that a given property name is unique across two different instantiations of the property
+     * Verifies that a given property name is the same across two different instantiations of the property
```

Comment:
This is obviously a change - possibly to be documented as BC break, even if internal.